tests/__init__.py

```diff
@@ -5,3 +5,121 @@
+                    },
+                    {
+                         "rel": "data",
+                         "href": "https://api.thousandeyes.com/v6/net/bgp-metrics/2837820"
+                    }
+               ],
+               "monitors": [
+                    {
+                         "active": 1,
+                         "metricsAtStart": "Reachability: 100%",
+                         "metricsAtEnd": "",
+                         "monitorId": 7,
+                         "monitorName": "Sydney-1",
+                         "prefixId": 10140666,
+                         "prefix": "140.82.121.0/24",
+                         "dateStart": "2022-04-29 14:00:00",
+                         "permalink": "https://app.thousandeyes.com/alerts/list/?__a=333666&alertId=444444444&monitorId=7",
+                         "network": "Telstra Pty Ltd (AS 1221)"
+                    },
+                    {
+                         "active": 1,
+                         "metricsAtStart": "Reachability: 100%",
+                         "metricsAtEnd": "",
+                         "monitorId": 21,
+                         "monitorName": "Los Angeles, CA",
+                         "prefixId": 10140666,
+                         "prefix": "140.82.121.0/24",
+                         "dateStart": "2022-04-29 14:00:00",
+                         "permalink": "https://app.thousandeyes.com/alerts/list/?__a=333666&alertId=444444444&monitorId=21",
+                         "network": "California State University, Office of the Chancellor (AS 2152)"
+                    },
+                    {
+                         "active": 1,
+                         "metricsAtStart": "Reachability: 100%",
+                         "metricsAtEnd": "",
+                         "monitorId": 42,
+                         "monitorName": "Geneva",
+                         "prefixId": 10140666,
+                         "prefix": "140.82.121.0/24",
+                         "dateStart": "2022-04-29 14:00:00",
+                         "permalink": "https://app.thousandeyes.com/alerts/list/?__a=333666&alertId=444444444&monitorId=42",
+                         "network": "VTX Services SA (AS 12350)"
+                    }
+               ],
+               "permalink": "https://app.thousandeyes.com/alerts/list/?__a=333666&alertId=444444444",
+               "ruleExpression": "[(((prefixLengthIPv4 >= 16) && (prefixLengthIPv4 <= 32)) || ((prefixLengthIPv6 >= 32) && (prefixLengthIPv6 <= 128)))]((reachability > 10%))",
+               "ruleId": 555555,
+               "ruleName": "BGP rule",
+               "testId": 4443322,
+               "testName": "BGP test name",
+               "violationCount": 61,
+               "type": "BGP",
+               "severity": "MINOR"
+          }
+]
+}
+
+ALERT_LIST = {}
+ALERT_LIST["alert"]=[ALERT_WITH_AGENTS["alert"][0],ALERT_WITH_MONITORS["alert"][0]]
```

Comment:
Why creating a dict with on key and call variable a list. Just create a list
```suggestion
ALERT_LIST = [ALERT_WITH_AGENTS["alert"][0], ALERT_WITH_MONITORS["alert"][0]]
```

Reply:
suggested change does not work.
Alert list must be defined in "alert" key

Reply:
Ok then 
```suggestion
ALERT_LIST={"alert": [ALERT_WITH_AGENTS["alert"][0],ALERT_WITH_MONITORS["alert"][0]]}
```

Reply:
ok